ext/devcontainer/README.md

```diff
@@ -0,0 +1,19 @@
+# Azure Developer CLI (azure-dev-cli)
+
+Installs the [Azure Developer CLI](https://github.com/Azure/azure-dev) along with needed dependencies.
+
+## Example Usage
+
+```json
+"features": {
+        "ghcr.io/azure/azure-dev/azure-dev-cli:1": {
+            "version": "latest"
+        }
```

Comment:
Does the path here follow the org/repo-name convention?  Thinking azure/azure-dev would be nice.

Reply:
The path here is where the container image for the feature lives. I've "mocked" it as though it's published to GitHub Packages, but we could use ACR if that's preferred

Reply:
let's go with azure/azure-dev/azd

Reply:
```suggestion
        "ghcr.io/azure/azure-dev/azd:1": {
            "version": "latest"
        }
```